### ISSUE TITLE
Cannot insert dublicate key

### ISSUE THREAD

**Issue by Almeonamy** [sergeysyrovatchenko/SQLIndexManager#13]
Hello. When I try to scan DB' indexes I get the error.
![image](https://user-images.githubusercontent.com/5328056/66985625-6559b400-f0e7-11e9-8af4-f1feaecaef7a.png)

**Comment by sergeysyrovatchenko** [comment]
Hi. Could you please provide version of your SQL Server and full DDL definition for this object (include all indexes, partitions schemas, etc)

```
USE ...
SELECT [name]
FROM sys.objects
WHERE [object_id] = 1886629764
```

**Comment by sergeysyrovatchenko** [comment]
Thanks for your feedback. Bugfix in progress.
Please provide result for this query:

```
SET NOCOUNT ON

IF OBJECT_ID('tempdb.dbo.#AllocationUnits') IS NOT NULL
    DROP TABLE #AllocationUnits

CREATE TABLE #AllocationUnits (
      ContainerID   BIGINT PRIMARY KEY
    , ReservedPages BIGINT NOT NULL
    , UsedPages     BIGINT NOT NULL
)

INSERT INTO #AllocationUnits (ContainerID, ReservedPages, UsedPages)
SELECT [container_id]
     , SUM([total_pages])
     , SUM([used_pages])
FROM sys.allocation_units WITH(NOLOCK)
GROUP BY [container_id]

IF OBJECT_ID('tempdb.dbo.#Partitions') IS NOT NULL
    DROP TABLE #Partitions

SELECT [object_id]
     , [index_id]
     , [partition_id]
     , [partition_number]
     , [rows]
     , [data_compression]
INTO #Partitions
FROM sys.partitions WITH(NOLOCK)
WHERE [object_id] > 255
    AND [rows] > 0

IF EXISTS(SELECT * FROM #Partitions GROUP BY [partition_id] HAVING COUNT(*) > 1)
    SELECT 'dublicates in #Partitions'

SELECT *
FROM (
    SELECT i.[object_id]
         , i.index_id
         , p.[partition_number]
         , p.[partition_id]
         , i.[name]
         , cnt = COUNT(*) OVER (PARTITION BY i.[object_id], i.index_id, p.[partition_number])
    FROM #AllocationUnits a
    JOIN #Partitions p ON a.ContainerID = p.[partition_id]
    JOIN sys.indexes i WITH(NOLOCK) ON i.[object_id] = p.[object_id] AND p.[index_id] = i.[index_id] 
    WHERE i.[type] IN (0, 1, 2, 5, 6)
        AND i.[object_id] > 255
) t
WHERE t.cnt > 1
```

**Comment by Almeonamy** [comment]
Result is empty
![image](https://user-images.githubusercontent.com/5328056/67074348-1d07c800-f1b3-11e9-95f8-476cb82e63b9.png)

**Comment by Almeonamy** [comment]
[result.txt](https://github.com/sergeysyrovatchenko/SQLIndexManager/files/3742719/result.txt)

**Comment by sergeysyrovatchenko** [comment]
Thanks. Will provide test build on this weekends.

**Comment by sergeysyrovatchenko** [comment]
Sorry for a late response. Fixed in v1.0.0.53. Thanks for your feedback.
